internal/charm/hooks/hooks.go

```diff
@@ -141,7 +141,7 @@ func (kind Kind) IsStorage() bool {
 // IsWorkload returns whether the Kind represents a workload hook.
 func (kind Kind) IsWorkload() bool {
 	switch kind {
-	case PebbleChangeUpdated, PebbleCustomNotice, PebbleReady:
+	case PebbleCustomNotice, PebbleReady:
```

Comment:
Need `PebbleCheckFailed` and `PebbleCheckRecovered`.
See https://github.com/juju/charm/commit/c09ff56304f49b8d98af09ea3a568f8547bdfc5d

Reply:
Done & Done